Core/Core/Configuration/Config/Config.swift

```diff
@@ -25,6 +25,8 @@ public protocol ConfigProtocol {
     var theme: ThemeConfig { get }
     var uiComponents: UIComponentsConfig { get }
     var discovery: DiscoveryConfig { get }
+    var program: DiscoveryConfig { get }
```

Comment:
@saeedbashir  If Discovery Config now contains both Discovery and Program config, perhaps we need to rename it to some "level up" entry. Thoughts?